src/screens/Settings/General/Region.js

```diff
@@ -0,0 +1,37 @@
+/* @flow */
+import { connect } from "react-redux";
+import _ from "lodash";
```

Comment:
```suggestion
import upperFirst from "lodash/upperFirst";
```
I don't know if we're doing it somewhere else in the app (which might make that not an issue here) but you could propably just import `upperFirst` to allow for tree shaking.

Reply:
Oh good catch thanks 👍

Reply:
Is there a fundamental difference between doing what you suggested and `import { upperFirst } from "lodash/upperFirst"` regarding tree shaking ?

Reply:
Ah ok good to know, we'll have to do some cleanup as some point then I think regarding this because in some other in the codebase we use a default import for lodash...
Thanks for the review!